### PR TITLE
Updated the README

### PR DESCRIPTION
**Pull request by corbinwunderlich**

### CODE REVIEW

**Review by rolandwunderlich — CHANGES_REQUESTED** [review]
Recommend some improvements to .gitignore

File: .gitignore
[the remark this answers is not in the document]
Comment (nousername33) [reply .gitignore:1]: Fixed

[the remark this answers is not in the document]
Comment (CorbinWunderlich) [reply .gitignore:1]: Whoops wrong account

[the remark this answers is not in the document]
Comment (CorbinWunderlich) [reply .gitignore:1]: Fixed

**Review by rolandwunderlich — APPROVED** [review]
Squash and merge it! https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/about-merge-methods-on-github#squashing-your-merge-commits

**Pull request merged**